test/solver_test.cpp

```diff
@@ -297,10 +297,10 @@ TEST(LARGE, 4) {
   EXPECT_EQ(dfpn.formatResult().substr(0, 7), "W 7 9 3");
 }
 
-}  // namespace dfpn
```

Comment:
all other looks good so far!

but are you sure about the changes here

Reply:
This one looks a little bit weird. I'll fix it in the next commit.